core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -901,7 +901,7 @@ protected function generateGlobalButtons()
 			// Custom icon (see #5541)
 			if ($v['icon'] ?? null)
 			{
-				$v['class'] = trim($v['class'] . ' header_icon');
+				$v['class'] = trim($v['class'] ?? '' . ' header_icon');
```

Comment:
```suggestion
				$v['class'] = trim(($v['class'] ?? '') . ' header_icon');
```

see https://3v4l.org/5AQjI#v8.0.9

Reply:
🙈 thanks! fixed in 08effd89b.